docker/py3-primary/chromium/download_chromedriver.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+# exit on errors
+set -e
+
+###########################
+# Script to download relevant chromedriver according to installed chrome version
```

Comment:
wdym installed chrome version? Can we use this for chromium image too?

Reply:
its taken directly from chromium image, its the same script.

it downloads the corresponding chromedriver according to the google-version installed